database/postgres/scripts/init.sql

```diff
@@ -0,0 +1,23 @@
+DROP TABLE IF EXISTS Places;
```

Comment:
Didn't this get replaced by Parcels?

Reply:
I've removed the script as it is irrelevant now.